libr/core/linux_heap_glibc.c

```diff
@@ -10,15 +10,18 @@
 #undef GH
 #undef GHT
 #undef GHT_MAX
+#undef r_read_le
```

Comment:
why redefine an r2-wise macro for something local? use a different name like `read_le` instead

Reply:
done